gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -108,6 +117,7 @@
   private HomeScreenViewModel viewModel;
   private MapContainerFragment mapContainerFragment;
   private BottomSheetBehavior<View> bottomSheetBehavior;
+  private PublishSubject<FragmentManager> showFeatureDialogRequests;
```

Comment:
`FragmentManager` doesn't seem like the kind of thing we'd want to stream; onCreate() has access to it as well, so can we just pass Object or clicked View here instead?

Reply:
Yeah, that makes a lot more sense lol. Please double check that I'm passing a dummy Object through correctly--not super familiar with OOP as it is nor with this technique.